internal/transport/grpchttp2/framer.go

```diff
@@ -242,6 +246,8 @@ func (f *WindowUpdateFrame) Header() *FrameHeader {
 	return f.hdr
 }
 
+func (f *WindowUpdateFrame) Free() {}
```

Comment:
Maybe this needs to be rebased after the other commit where you added docstrings to all these methods?

Reply:
Yes, I rebased to master and was waiting for the other changes to be merged. I'm going to merge those changes into this branch as well.